spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -532,30 +532,70 @@
       # a <saml:Conditions> element, which gives the conditions under which
       # the assertion is to be considered valid
       context 'Conditions' do
```

Comment:
I think "describe" would make more sense here and elsewhere in the file (not related to this PR I guess)